### PR TITLE
Added search results "pagination"

### PR DESCRIPTION
**Pull request by MaxLiebe**
Added a "get more results" button to the ResultsWrapper so one can search for more than just 20 results with a single query.

Technically not proper pagination, but I believe it serves its purpose well enough.

### CODE REVIEW

File: client/src/components/CurrentQueue.vue [hunk 1]
```diff
@@ -137,6 +147,20 @@ async function removeFromQueue(video) {
   });
 }
 
+async function clearQueue() {
+  if (!props.admin) return;
+  //remove all videos from the queue
+  const data = await new Promise((resolve) => {
+    socket.emit("clear-queue", null, (callback) => {
```
Comment (Menkveld-24) [inline client/src/components/CurrentQueue.vue:154]: Unnecessary parameter '_'

File: client/src/components/ResultsWrapper.vue [hunk 2]
```diff
@@ -36,6 +42,7 @@ import ContentField from "@/layout/ContentField.vue";
 
 defineProps({
   videos: Object,
+  continuationToken: String,
```
Comment (Menkveld-24) [inline client/src/components/ResultsWrapper.vue:45]: Unused prop

**Review by Menkveld-24 — COMMENTED** [review]
Nice!
I would maybe align the "pagination" button to the bottom-right as that feels a bit more natural to me

**Comment by Jessseee** [comment]
Changes moved to #49.

**Pull request closed without merging**